apis/cloud.redhat.com/v1alpha1/clowdjobinvocation_types.go

```diff
@@ -76,9 +76,17 @@ type IqeJobSpec struct {
 	TestImportance *[]string `json:"testImportance,omitempty"`
 }
 
-type UiSpec struct {
-	// Indiciates the presence of a selenium container
-	Enabled bool `json:"enabled"`
```

Comment:
Since `ui.enabled` didn't do anything it wasn't really used by anyone anywhere as far as I know... for example it was not used in bonfire's CJI template: https://github.com/RedHatInsights/bonfire/blob/master/bonfire/resources/default-iqe-cji.yaml

I can see it is also not used in cloudigrade's release-gating CJI template: https://github.com/cloudigrade/cloudigrade/blob/master/deployment/stage-smoke.yaml

I think just completely removing it should be ok.

Reply:
While it should be - I would prefer we follow our procedure and deprecate it - especially as we are not trying to reuse it

Reply:
OK, it literally did nothing previously, so I suppose I can just leave it in the spec but update the comment to say it is not used?

Reply:
Added 'Enabled' into the IqeUISpec